api/src/main/java/org/ccci/idm/user/DefaultUserManager.java

```diff
@@ -34,12 +36,20 @@
     private static final EmailValidator VALIDATOR_EMAIL = EmailValidator.getInstance();
 
     @NotNull
+    @Autowired(required = false)
```

Comment:
Is it not possible to leave off the `@Autowired` annotation altogether and simply rely on XML file bean definition/injection when required by the application in question?

On that note, I suppose some of our applications which explicitly inject `userDao` by way of XML bean definition need no longer do so due to the `@Inject` annotation. Would that be correct?